ui/src/components/RecordingSection.js

```diff
@@ -52,11 +52,41 @@ export class RecordingSection extends LitElement {
                     .audioStream=${this.audioStream}
                     .context=${this.context}
                 >
-                </vox-sound-wave>`;
+                </vox-sound-wave>
+                <p>w</p>`;
+            case CollectionStates.NOT_RECORDING:
```

Comment:
this isnt documentation, intended to go with this pull request?

Reply:
Lol no. I noticed and fixed it